test/index.ts

```diff
@@ -1,12 +1,11 @@
 import { expect } from "chai";
-import { Chart } from "chart.js";
-
 import ChartUtils from "../src/index.ts";
+import sinon from "sinon";
 
 describe("Testing ChartUtils", () => {
   it("Testing afterDatasetsUpdate", () => {
 
-    const _meta = {
+    let meta = {
```

Comment:
this can stay a const, you are not changing the value(i.e. pointer), you are changing values within.